### PR TITLE
Splash Screen added

### PR DESCRIPTION
**Pull request by rob729**

### CODE REVIEW

**Comment by rob729** [comment]
@sridharjajoo This PR is for #17 issue. 
Please review it.

File: app/src/main/java/com/example/sridharjajoo/newsapp/SplashScreen.java [hunk 1]
```diff
@@ -0,0 +1,17 @@
...
+import android.content.Intent;
+import android.support.v7.app.AppCompatActivity;
+import android.os.Bundle;
+
+public class SplashScreen extends AppCompatActivity {
+
+    @Override
+    protected void onCreate(Bundle savedInstanceState) {
+        super.onCreate(savedInstanceState);
+        Intent intent = new Intent(this,NewsMainActivity.class);
```
Comment (sridharjajoo) [inline app/src/main/java/com/example/sridharjajoo/newsapp/SplashScreen.java]: space between , and NewsMainActivity.class

File: app/src/main/java/com/example/sridharjajoo/newsapp/SplashScreen.java [hunk 2]
```diff
@@ -0,0 +1,17 @@
...
+        startActivity(intent);
+        finish();
+
```
Comment (sridharjajoo) [inline app/src/main/java/com/example/sridharjajoo/newsapp/SplashScreen.java]: remove extra line

File: app/src/main/res/values/styles.xml [hunk 3]
```diff
@@ -8,4 +8,9 @@
         <item name="colorAccent">@color/colorAccent</item>
     </style>
 
+    <style name="SplashTheme" parent="Theme.AppCompat.NoActionBar">
+        <item name="android:windowBackground">@drawable/splash_layer</item>
+    </style>
+
+
```
Comment (sridharjajoo) [inline app/src/main/res/values/styles.xml]: remove extra line

**Comment by sridharjajoo** [comment]
Also why is the progress bar invisible when the NewsMainActivity is being loaded?

**Comment by rob729** [comment]
@sridharjajoo I also don't know why is that happening. 
What should I do about it?

**Comment by rob729** [comment]
I have done the other recommended changes.

**Comment by sridharjajoo** [comment]
@rob729 Try to find the solution for the same

**Pull request closed without merging**